Travel Scheduler/View Controllers/Place.m

```diff
@@ -201,48 +203,74 @@ -(NSMutableArray *)getRestaurantsPeriodsArrayFromOpeningTime:(float)openingTime
     if(closingTime >= 19) {
         [arrayOfPeriods addObject:@(evening)];
     }
-    
     return arrayOfPeriods;
 }
 
-#pragma mark - Image methods
-- (void)setImageViewOfPlace:(Place *)myPlace withPriority:(bool)priority withDispatch:(dispatch_semaphore_t)setUpCompleted withCompletion:(void (^)(UIImage *image, NSError *error))completion{
-    dispatch_async(dispatch_get_main_queue(), ^{
-        GMSPlaceField fields = (GMSPlaceFieldPhotos);
-        
-        if(priority) {
-            dispatch_semaphore_signal(setUpCompleted);
+#pragma mark - Methods to get the array of nearby places (for hubs only)
+-(void)createDictionaryOfArrays{
+    NSArray *arrayOfTypes = [[NSArray alloc]initWithObjects:@"lodging", @"restaurant", @"museum", @"park", nil];
+    for(NSString *type in arrayOfTypes){
+        dispatch_semaphore_t createdTheArray = dispatch_semaphore_create(0);
+        [self makeArrayOfNearbyPlacesWithType:type withCompletion:^(bool success, NSError * _Nonnull error) {
+            if(success) {
+                NSLog(@"so far so good");
```

Comment:
Could consider removing NSLogs. I know these were good for debugging but I think they are contributing to the time it takes for code to get arrays and making app slow.

Reply:
Good point! I think for now I am just keeping them to keep track of the places where we should put code to deal with possible problems (such as showing a pop up when there is a network error, for example). I will try to get to that today and remove them! :-)